src/test/java/de/adorsys/keycloak/config/service/ImportManagedClientsFullIT.java

```diff
@@ -0,0 +1,56 @@
+package de.adorsys.keycloak.config.service;
+
+import de.adorsys.keycloak.config.AbstractImportTest;
+import org.junit.jupiter.api.Test;
+import org.keycloak.representations.idm.ClientRepresentation;
+import org.keycloak.representations.idm.RealmRepresentation;
+import org.springframework.test.context.TestPropertySource;
+
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.core.Is.is;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@TestPropertySource(properties = {
+        "import.managed.client=full"
+})
+public class ImportManagedClientsFullIT extends AbstractImportTest {
+    private static final String REALM_NAME = "realmWithClients";
```

Comment:
Please use an other realm name here. Each realm should be unique for every test class to avoid race conditions.